addons/website_forum/static/src/js/website_forum.js

```diff
@@ -248,35 +429,37 @@ odoo.define('website_forum.website_forum', function (require) {
                 $('button.validation_email_close').click();
             }
         });
-    });
-
-    $('.validated_email_close').on('click', function (ev) {
+    },
+    /**
+     * @private
+     */
+    _onCloseValidatedEmail: function () {
         ajax.jsonRpc("/forum/validate_email/close", 'call', {});
```

Comment:
This should have been converted this `this._rpc` too ;)

Reply:
Indeed, by the way I'll have a look on how you convert those in this._rpc because I don't fully understand how you handle args in some situations